manifests/base/suite-runner-template.yaml

```diff
@@ -58,13 +58,19 @@ data:
                 name: {etos_configmap}
             - secretRef:
                 name: {etos_rabbitmq_secret}
+            - configMapRef:
+                name: etos-observability
```

Comment:
We should not hard-code this configmap. This sort of thing should now be configured by those deploying ETOS.

Reply:
Changed it to a variable, similar to etos_configmap.